src/imports/logger/ErrorsTransport.js

```diff
@@ -0,0 +1,77 @@
+//import Rollbar from 'rollbar'
+
+import * as Sentry from '@sentry/node'
+import Transport from 'winston-transport'
+import { SPLAT } from 'triple-beam'
+import { forEach } from 'lodash'
+import Config from '../../server/server.config'
+
+const { env, sentryDSN, version, network } = Config
+const logEnvAllowed = !['test', 'development'].includes(env)
```

Comment:
let's add env var for this so we could just add it to the environments where it's needed